packages/client/src/client.ts

```diff
@@ -1778,6 +1776,8 @@ export class SkipClient {
     gasAmountMultiplier?: number;
     getFallbackGasAmount?: clientTypes.GetFallbackGasAmount;
   }) {
+    // tx index -> gas token used
+    let gasTokenUsed: Record<number, Coin> = {}
```

Comment:
@codingki let's update the name of this one to gasTokenRecord as well, and should they both be Record<number, Coin> ?

Reply:
updated, yes it should be a record. because not all tx is a `cosmosTx` so it won't be always sorted, can be random index

Reply:
@codingki I understand we want it to be a record I just mean the other one is `Record<string, Coin>` instead of `Record<number, Coin>`